engines/engine.h

```diff
@@ -446,61 +558,80 @@ class Engine {
 	/**
 	 * Set the game time counter to the specified time.
 	 *
-	 * This can be used to set the play time counter after loading a savegame
-	 * for example. Another use case is in case the engine wants to exclude
-	 * time from the counter the user spent in original engine dialogs.
+	 * This can be used to set the play time counter after loading a savegame,
+	 * for example. Another use case is when the engine wants to exclude from
+	 * the counter the time that the user spent in original engine dialogs.
 	 *
 	 * @param time Play time to set up in ms.
 	 */
 	void setTotalPlayTime(uint32 time = 0);
 
+	/**
+	 * Return the TimerManager instance used by this engine.
+	 */
 	inline Common::TimerManager *getTimerManager() { return _timer; }
+	/**
+	 * Return the EventManager instance used by this engine.
+	 */
 	inline Common::EventManager *getEventManager() { return _eventMan; }
+	/**
+	 * Return the SaveFileManager instance used by this engine.
+	 */
 	inline Common::SaveFileManager *getSaveFileManager() { return _saveFileMan; }
 
 public:
-	/** On some systems, check if the game appears to be run from CD. */
+	/** On some systems, check whether the game appears to be run from CD. */
 	void checkCD();
 
 
 	/**
-	 * Checks for whether it's time to do an autosave, and if so, does it.
+	 * Check whether it is time to autosave, and if so, do it.
 	 */
 	void handleAutoSave();
 
 	/**
-	 * Does an autosave immediately if autosaves are turned on
+	 * Autosave immediately if autosaves are enabled.
 	 */
 	void saveAutosaveIfEnabled();
 
 	/**
-	 * Indicates whether an autosave can currently be saved.
+	 * Indicate whether an autosave can currently be done.
 	 */
 	virtual bool canSaveAutosaveCurrently() {
 		return canSaveGameStateCurrently();
 	}
 
 	/**
-	 * Returns the slot that should be used for autosaves
-	 * @note	This should match the meta engine getAutosaveSlot() method
+	 * Return the slot that should be used for autosaves.
+	 *
+	 * @note	This should match the meta engine getAutosaveSlot() method.
 	 */
 	virtual int getAutosaveSlot() const {
 		return 0;
 	}
-
+	
+	/**
+	 * Check whether it is time to autosave based on the
+	 * provided @p lastSaveTime. @todo Doc check please
+	 */
```

Comment:
This is correct but we should indicate that this function is deprecated.
Now autosaves are handled directly by the `Engine` class and derived classes do not need to worry about it other than to implement `canSaveAutosaveCurrently()` and `getAutosaveSlot()` if the default implementations are not sufficient.